doc/classes/RigidBody2D.xml

```diff
@@ -1,14 +1,17 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="RigidBody2D" inherits="PhysicsBody2D" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
-		Physics Body whose position is determined through physics simulation in 2D space.
+		A body that is controlled by the 2D physics engine.
 	</brief_description>
 	<description>
-		This is the node that implements full 2D physics. This means that you do not control a RigidBody2D directly. Instead you can apply forces to it (gravity, impulses, etc.), and the physics simulation will calculate the resulting movement, collision, bouncing, rotating, etc.
-		This node can use custom force integration, for writing complex physics motion behavior per node.
-		This node can shift state between regular Rigid body, Kinematic, Character or Static.
-		Character mode forbids this node from being rotated.
-		As a warning, don't change RigidBody2D's position every frame or very often. Sporadic changes work fine, but physics runs at a different granularity (fixed hz) than usual rendering (process callback) and maybe even in a separate thread, so changing this from a process loop will yield strange behavior.
+		This is the node that implements simulated 2D physics. You do not control a RigidBody2D directly. Instead you apply forces to it (gravity, impulses, etc.) and the physics simulation calculates the resulting movement based on its mass, friction, and other physical properties.
+		A RigidBody2D has 4 behavior modes (see [member mode]):
+		- [b]Rigid[/b]: The body behaves as a physical object.  It collides with other bodies and responds to forces applied to it. This is the default mode.
```

Comment:
double spaces after first sentence.

Reply:
I have the hardest time breaking that habit.